src/functionalTest/resources/application-functional.yaml

```diff
@@ -1,14 +1,14 @@
-targetInstance: ${TEST_URL:http://localhost:8090}
+targetInstance: ${TEST_URL:http://rd-professional-api-pr-860.service.core-compute-preview.internal}
 
 s2s-url: ${S2S_URL_FOR_TESTS:http://rpe-service-auth-provider-aat.service.core-compute-aat.internal}
 
 s2s-name: rd_professional_api
 
 idam.api.url: ${IDAM_URL:https://idam-api.aat.platform.hmcts.net}
-oauth2-client-secret: ${OAUTH2_CLIENT_SECRET:AAAAAAAAAAAAAAAA}
-idam.auth.clientId: ${OAUTH2_CLIENT_ID:AAAAAAAAAAAAAAAA}
+oauth2-client-secret: ${OAUTH2_CLIENT_SECRET:cc5f2a6-9690-11e9-bc42-526af7764f64}
+idam.auth.clientId: ${OAUTH2_CLIENT_ID:rd-professional-api}
```

Comment:
seems this is hardcoded to use some actual secret now..

Reply:
yes ..will remove that